### PR TITLE
Speculative fix for Mac to avoid zombie "atos" processes.

### PR DESCRIPTION
**Pull request by Dor1s**

### CODE REVIEW

File: src/python/crash_analysis/stack_parsing/stack_symbolizer.py [hunk 1]
```diff
@@ -382,23 +382,25 @@ def symbolize(self, addr, binary, offset):
...
+      if match:
+        function_name = match.group(1)
+        function_name = re.sub('\(.*?\)', '', function_name)
+        file_name = match.group(3)
+        return [get_stack_frame(binary, addr, function_name, file_name)]
+      else:
+        return ['%s in %s' % (addr, atos_line)]
+    except Exception:
+      logs.log_error(
+          '%s %s\n%s' % (binary, str(offset), traceback.format_exc(15)))
```
Comment (inferno-chromium) [inline src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: please remove traceback.format_exc(15) and its %s from all instances in this file. log_error automatically logs stacktraces, so this is unneeded. this is very old code, so we needed it in old days.

Comment (inferno-chromium) [reply src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: Maybe even starting message like Symbolization failed for atos or another place Symbolization failed for llvm-symbolizer for the other similar instance.

Comment (Dor1s) [reply src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: Done.

File: src/python/crash_analysis/stack_parsing/stack_symbolizer.py [hunk 2]
```diff
@@ -289,7 +288,8 @@ def symbolize(self, addr, binary, offset):
         result.append(get_stack_frame(binary, addr, function_name, file_name))
 
     except Exception:
-      logs.log_error('%s\n%s' % (symbolizer_input, traceback.format_exc(15)))
+      logs.log_error('Symboization using llvm-symbolizer failed for: "%s".' %
```
Comment (inferno-chromium) [inline src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: nit: typo Symboization

File: src/python/crash_analysis/stack_parsing/stack_symbolizer.py [hunk 3]
```diff
@@ -382,23 +382,25 @@ def symbolize(self, addr, binary, offset):
...
+      match = re.match('^(.*) \(in (.*)\) \((.*:\d*)\)$', atos_line)
+      if match:
+        function_name = match.group(1)
+        function_name = re.sub('\(.*?\)', '', function_name)
+        file_name = match.group(3)
+        return [get_stack_frame(binary, addr, function_name, file_name)]
+      else:
+        return ['%s in %s' % (addr, atos_line)]
+    except Exception:
+      logs.log_error('Symbolization using atos failed for "%s %s".' %
```
Comment (inferno-chromium) [inline src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: nit: for: (colon after for)

File: src/python/crash_analysis/stack_parsing/stack_symbolizer.py (hunk 2 above)
Comment (Dor1s) [reply src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: Oops, thanks! Done.

File: src/python/crash_analysis/stack_parsing/stack_symbolizer.py (hunk 3 above)
Comment (Dor1s) [reply src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: Done

**Comment by Dor1s** [comment]
`run_all_tests` passed locally, merging.

**Pull request merged**